fpga/src/main/scala/genesys2/IOBinders.scala

```diff
@@ -0,0 +1,74 @@
+package chipyard.fpga.genesys2
+
+import chipyard.CanHaveMasterTLMemPort
+import chipyard.iobinders.{GetSystemParameters, OverrideIOBinder, OverrideLazyIOBinder}
+import chisel3._
+import chisel3.experimental.{DataMirror, IO}
+import freechips.rocketchip.devices.debug._
+import freechips.rocketchip.diplomacy.InModuleBody
+import freechips.rocketchip.prci.{ClockSinkNode, ClockSinkParameters}
+import freechips.rocketchip.subsystem.BaseSubsystem
+import freechips.rocketchip.tilelink.TLBundle
+import freechips.rocketchip.util.{HeterogeneousBag, PSDTestMode}
+import sifive.blocks.devices.uart.HasPeripheryUARTModuleImp
+
+class WithUARTIOPassthrough extends OverrideIOBinder({
```

Comment:
This code seems like a copy of the VCU118 code. Maybe we can dedup this by creating a folder like: `fpga/src/main/scala/common` and putting both sources there (with a new package name `chipyard.fpga.common`). Then we just reference that.

Reply:
Good idea, I noticed I was just copying and pasting code when porting too 😂. Should I try to do some dedup work now or after this PR get merged?

Reply:
I would wait until we get back to you on the testing side and `fpga-shells` gets merged.